roles/neutron-common/templates/usr/local/bin/neutron-reset

```diff
@@ -256,6 +272,7 @@ class ResetNeutron(object):
             if was_running:
                 running_agents.append(agent)
 
+        interfaces = Interface.enumerate()
```

Comment:
`Interfaces.enumerate()` was a generator, if you want to save the tap interfaces it would be better to filter that generator into a list of tap interfaces;
```nova_tap_interfaces = [x for x in Interface.enumerate() if not x.is_neutron_interface()]```